server/fishtest/templates/tests_view.mak

```diff
@@ -847,6 +847,12 @@
           dots = 3; // fall back to the three dot diff request as the diff will be rebased
       % endif
     % else:
+      % if (
```

Comment:
mako %if written in only one line.

If I'm not wrong, it does not work

![image](https://github.com/official-stockfish/fishtest/assets/15718418/6a3e36d5-254f-4fd9-839d-c82c04935c64)

Reply:
Sorry I can't test the PR rn, I changed it to one line I'll check if that works when home.